pages/moderation.jsx

```diff
@@ -28,7 +28,7 @@ class Moderation extends React.Component {
   }
 
   render() {
-    if (!user.moderator) return <NotFound />;
+    //if (!user.moderator) return <NotFound />;
```

Comment:
Is there a way to view moderation without commenting that out?...that was the only way I could view it 😅

Reply:
Yea if your account has moderator rights... let me check with Giddy and ask you to the group. On localhost you should be able to do that yourself since you are the superuser.